### PR TITLE
Game updates

### PR DESCRIPTION
**Pull request by athompsonScottLogic**

### CODE REVIEW

File: whichone/static/js/game.js [hunk 1]
```diff
@@ -173,13 +173,23 @@ function compareArtists() {
     } else {
         artistList = user.top_artists;
         numTracks = artistList.length;
+
+        oldNum1 = numTracks[document.getElementById("text1a").textContent];
+        oldNum2 = numTracks[document.getElementById("text2a").textContent];
+
+        console.log(oldNum1, oldNum2);
```
Comment (connor-settle) [inline whichone/static/js/game.js]: Remove console logs

File: whichone/static/js/game.js [hunk 2]
```diff
@@ -173,13 +173,23 @@ function compareArtists() {
...
+
         num1 = Math.floor(Math.random() * numTracks);
 
+        while (num1 == oldNum1 || num1 == oldNum2) {
```
Comment (connor-settle) [inline whichone/static/js/game.js]: Do you really need a while loop here and below? 😄

File: whichone/static/js/game.js (hunk 1 above)
Comment (athompsonScottLogic) [reply whichone/static/js/game.js]: done

File: whichone/static/js/game.js (hunk 2 above)
Comment (athompsonScottLogic) [reply whichone/static/js/game.js]: it makes it clear when each number is assigned, num1 needs to be defined in order t ocheck against num2. To be honest, the first while loop only runs around 2% of the time and the second one only runs once the vast majority of the time

**Review by PHardySL — APPROVED** [review]
happy with this

**Review by mcgill-a — APPROVED** [review]

**Pull request merged**